docs/openapi/components/schemas/credentials/EInvoiceCredential.yml

```diff
@@ -0,0 +1,1048 @@
+$linkedData:
+  term: EInvoiceCredential
+  '@id': https://w3id.org/traceability#EInvoiceCredential
+title: e-Invoice Credential
```

Comment:
This is based on specific Mexican requirements, I believe? I suggest we  reflected that from the title and description.